test/e2e/environment.go

```diff
@@ -91,7 +91,8 @@ type TestEnvironment struct {
 type TestEnvOption func(opt *testEnvOptions)
 
 type testEnvOptions struct {
-	Image string
+	Image               string
+	InstallViaKustomize bool
```

Comment:
How about leaving here a comment explaining that if we define this to be `false` then no operator will be installed and it's a particular test's responsibility to deploy it in another way (e.g. with Helm)?

Reply:
👍 Ideally we'd change the whole "framework" that we've got but the usage with helm and kustomize is so different (values vs no values etc) that at this stage I believe there's no point.

Added the comment.